packages/rulesets/src/spectral/functions/properties-type-object-no-definition.ts

```diff
@@ -0,0 +1,21 @@
+// If a property is defined with `'type': 'object'`, it must also define an object body.
+// RPC Code: RPC-Policy-V1-03
+const errorMessageObject =
+  "Properties with type:object that dont reference a model definition are not allowed. ARM doesnt allow generic type definitions as this leads to bad customer experience."
+const errorMessageNull = "Properties with type NULL is not allowed."
```

Comment:
>is [](http://example.com/codeflow?start=52&length=2)

are

Reply:
Properties with type NULL are not allowed. Either specify the type as object and reference a model or specify a primitive type